integration_tests/kpi.feature

```diff
@@ -53,16 +53,24 @@ Feature: KPI integration tests
      And the penalties of the 'Penalty PV' is the sum of the residual available energy
      And the penalty cost of the 'Penalty PV' is respecting the penalty rate
 
-  Scenario: D3ASIM-2103, self sufficiency reports correct values on levels higher than houses
-     Given we have a scenario named kpi.d3asim_2103
+  Scenario Outline: D3ASIM-2103, self sufficiency reports correct values on levels higher than houses
+     Given we have a scenario named <scenario_file>
      And d3a is installed
-     When we run the d3a simulation on console with kpi.d3asim_2103 for 24 hrs
+     When we run the d3a simulation on console with <scenario_file> for 24 hrs (30, 30)
```

Comment:
Answered already here https://github.com/gridsingularity/d3a/pull/709#discussion_r403036674

Reply:
What I meant is since we have used 60secs as tick_length & 60mins as slot_length for almost all of our integration_tests, so I was wondering if you have observed any change in KPI results with a change in `tick_length & slot_length` ?

Reply:
Yes, with 60 seconds tick and slot length no trades take place. There are not enough ticks for the offer / bids to match